cni/plugin.go

```diff
@@ -161,14 +161,14 @@ func (plugin *Plugin) InitializeKeyValueStore(config *common.PluginConfig) error
 		var err error
 		plugin.Store, err = store.NewJsonFileStore(platform.CNIRuntimePath + plugin.Name + ".json")
 		if err != nil {
-			log.Printf("[cni] Failed to create store, err:%v.", err)
+			log.Printf("[cni] Failed to create store: %v.", err)
```

Comment:
@jhowardmsft - I know its not you but typically no punctuation across other golang projects.

Reply:
Also. Typically the "[cni]" portion would be done via a logrus context or equivalent. (Again just FYI)

Reply:
Yeah, but I'm not changing the world here :)